src/components/MultiHopTrade/components/TradeInput/components/TradeQuotes/TradeQuotes.tsx

```diff
@@ -17,103 +21,124 @@ type TradeQuotesProps = {
 
 const arrowDownIcon = <ArrowDownIcon />
 
-export const TradeQuotes: React.FC<TradeQuotesProps> = memo(({ sortedQuotes, isLoading }) => {
-  const activeQuoteIndex = useAppSelector(selectActiveQuoteIndex)
-  const translate = useTranslate()
-  const [showAll, setShowAll] = useState(false)
-  const bestQuoteData = sortedQuotes[0]
-  const bottomOverlay = useColorModeValue(
-    'linear-gradient(to bottom,  rgba(255,255,255,0) 0%,rgba(255,255,255,0.4) 100%)',
-    'linear-gradient(to bottom,  rgba(24,27,30,0) 0%,rgba(24,27,30,0.9) 100%)',
-  )
+export const TradeQuotes: React.FC<TradeQuotesProps> = memo(
+  ({ sortedQuotes: _sortedQuotes, isLoading }) => {
+    const wallet = useWallet().state.wallet
+    const { chainId: buyAssetChainId } = useAppSelector(selectBuyAsset)
+    const isSnapInstalled = useIsSnapInstalled()
+    const walletSupportsBuyAssetChain = useWalletSupportsChain({
+      chainId: buyAssetChainId,
+      wallet,
+      isSnapInstalled,
+    })
+    const isBuyAssetChainSupported = walletSupportsBuyAssetChain
 
-  const hasMoreThanOneQuote = useMemo(() => {
-    return sortedQuotes.length > 1
-  }, [sortedQuotes.length])
+    // TODO(gomes): buy chain not supported effectively means connecting MM/other EVM-only wallets and trying to get a THOR trade quote
+    // This makes things not borked as it doesn't display quotes at all vs. stale quotes with the wrong symbol, but we may want to ditch the
+    // destination param instead, to be able to get a quote, *if* and only if we can ensure proper guards are in place to never allow a memo without a destination
```

Comment:
In addition to this being a monkey patch rather than us displaying the quotes (which, as this TODO explains, may or may not be safe so we'll want to dedicate a PR with extra safety in place *if* we assess it 100% safe), the implementation in and by itself is also not the best way to go around this, but is the best way to go in fixing this immediate issue.

See the PR description:
> Because we leverage cached quote RTK queries, we never clear quotes anywhere in the app, which is usually not an issue as quotes will either

This is a mistake and another root cause of our stale issues, the first one being selected zero'th stale quotes, which has been fixed in https://github.com/shapeshift/web/pull/5809/commits/3d1669ce380d61bc586fda7c26f3366874dc58b1

The best way to go around this will be to properly invalidate `swappersApi.endpoints.getTradeQuote`:
- after our refetch interval passes, so we never keep stale quotes in memory, which  *always* become useless the moment we start refetching
- on args change

Doing so would mean storing less, and would fix this issue, though it has bigger swapper implications, so going to revert this PR and tackle it that way as a follow-up.